lua/neoclip/settings.lua

```diff
@@ -2,6 +2,7 @@ local M = {}
 
 local settings = {
     history = 1000,
+    db_path = nil,
```

Comment:
It would be nice if we can keep the default location of dbs in `vim.fn.stdpath("data") .. "/databases/neoclip.sqlite3"`

Reply:
Good point! Done in latest commit :+1: